app/assets/javascripts/wizard/components/homepage-preview.js

```diff
@@ -9,7 +9,12 @@ export default createPreviewComponent(659, 320, {
   logo: null,
   avatar: null,
 
-  @observes("step.fieldsById.homepage_style.value")
+  @observes(
```

Comment:
Since we're in here - can we instead trigger an action when the value changes in select kit and remove the observer?

Reply:
I think I've done this in https://github.com/discourse/discourse/pull/14132/commits/f2fdf9e8be49a9abe5f134fb60c76812df42fb95. It's still a little awkward, because I found it necessary to add a new property, `stylingDropdown`, that is passed down to the preview components. Let me know if this is not what you had in mind.